src/ucp/tag/offload.c

```diff
@@ -665,7 +666,7 @@ ucs_status_t ucp_tag_offload_start_rndv(ucp_request_t *sreq)
 
         /* RNDV will be performed by the SW - can register with SW RNDV lanes
          * to get multirail benefits */
-        status = ucp_rndv_reg_send_buffer(sreq);
+        status = ucp_rndv_reg_send_buffer(sreq, param);
```

Comment:
maybe can also reuse for tag offload flow?

Reply:
IMO better postpone it to protov2 and ucp rcache refactoring